### DISCUSSION TITLE
/include member never shows anything in Outline View

### DISCUSSION THREAD

**Discussion by mwmayer4** [codefori/vscode-ibmi#867] [General]
Using Code for IBM i v1.5.11. last updated 9/2/2022. WIth source member type SQLRPGLE or RPGLE, any _/include memberName_ never shows any contents in the Outline View. I saw a previous thread wherein nothing was showing for SQLRPGLE and that it was fixed (back in February 2022 maybe?).  For example, the following code shows nothing in the outline view but the dummy indicator field. 
`**FREE

ctl-opt dftactgrp(*no) actgrp('CODE4I')
        bnddir('YAJL')
        decedit('0.')
        option(*srcstmt: *nodebugio: *noshowcpy);
/include yajl_h
dcl-s dummy ind;
*INLR=*ON;`

It compiles, of course, and the listing shows all of the YAJL items associated with the /include. 
Any suggestions? Maybe the outline view isn't suppose to show it, but I would doubt that. 
Thanks!

**Comment by chrjorgensen** [comment]
@mwmayer4

First, I think this issue should be reported in the extension [vscode-rpgle](https://github.com/halcyon-tech/vscode-rpgle) which is the one providing the outline, linter etc.

Second, have you tried changing the current (home) directory to the one containing the source and include file? I think I've read in another issue in the other or this repo that the extension - due to performance issue - only looks for the include files in the current directory.

Hope you find a solution!

**Reply by mwmayer4** [reply]
I'll try posting there too. I'm somewhat new to the arena so I wasn't sure where I should post. Yes, I have an /include member in the current library as well. It doesn't pull it in either. Thanks!
